src/messaging/ReliableMessageMgr.cpp

```diff
@@ -137,6 +137,7 @@ void ReliableMessageMgr::ExecuteActions()
             return Loop::Continue;
         }
 
+        entry->sendCount++;
```

Comment:
It occurs to me that the spec is _extremely_ unclear whetheron whether the "transmission count of the message" used in the MRP backoff formula is the count _before_ the current send or _after_. @turon could we get that fixed, please?

But yes, now we compute the first backoff (from StartRetransmission) with n == 0, the next one with n == 1, etc, which seems like the spec's intent, I think....

Reply:
Yes, the spec intent is first timeout (starting after initial send) is count 0.  Spec issue # TBD.